apps/token/src/components/eth-wallet/eth-wallet.tsx

```diff
@@ -163,15 +163,13 @@ const ConnectedKey = () => {
         />
       )}
       <WalletCardActions>
-        <Link style={{ flex: 1 }} to={`${Routes.STAKING}/associate`}>
-          <Button variant="primary" className="w-full">
-            {t('associate')}
-          </Button>
+        <Link className="flex-1" to={`${Routes.STAKING}/associate`}>
+          <button className="w-full px-28 border h-28">{t('associate')}</button>
         </Link>
-        <Link style={{ flex: 1 }} to={`${Routes.STAKING}/disassociate`}>
-          <Button variant="primary" className="w-full">
+        <Link className="flex-1" to={`${Routes.STAKING}/disassociate`}>
+          <button className="w-full px-28 border h-28">
```

Comment:
i think buttons are not valid when nested under anchor elements. 

quote: "The [a](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element) element can be wrapped around entire paragraphs, lists, tables, and so forth, even entire sections, so long as there is no interactive content within (e.g., buttons or other links)" (ref: https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element)